libs/cms/src/lib/generated/contentfulTypes.d.ts

```diff
@@ -1123,6 +1135,15 @@ export interface ILinkFields {
 
   /** Link reference */
   linkReference?: IArticle | IArticleCategory | ILinkUrl | INews | undefined
+
+  /** Searchable */
+  searchable?: boolean | undefined
+
+  /** Search Description */
+  intro?: string | undefined
```

Comment:
I'm not 100% whether I should name this intro or description.

If named description then in the graphql search query we need to do
```
...on Link {
  id
  intro: description
}
```

anyway.

Reply:
I think you are right naming it `intro` is the correct thing to do